src/components/LeafletMap.js

```diff
@@ -233,6 +233,30 @@ class LeafletMap extends Component {
     return false;
   }
 
+  toFixedNoRounding = function(number, n) {
```

Comment:
Can you tell me what this function does? Can we make it clear by the name of the function?

Reply:
same as toFixed() in javascript, but no rounding at the end

Reply:
basically I want to turn all array of coordinates into strings and compare whether all markers are in the same cluster

Reply:
Thanks! See review for sugestion